docs/AKSScalarTerraformDeploymentGuide.md

```diff
@@ -31,6 +31,12 @@ You also need to have enough permissions to deploy the Kubernetes cluster with T
 * 1 Bastion instance with a public IP
 * 1 Monitor instance to collect metrics and logs
 
+## Clone scalar-teraform-examples
+
+```console
+$ git clone https://github.com/scalar-labs/scalar-terraform-examples -b <master or tag> modules
```

Comment:
I don't think it is the best way to clone another repository inside the repository. There would be no problem with cloning the examples outside of the `${SCALAR_K8S_HOME}`. How about introducing a new environment variable, `SCALAR_TERRAFORM_EXAMPLES`?

Reply:
Good point! 👍 

Fixed in 0999727